.github/CODEOWNERS

```diff
@@ -8,99 +8,760 @@
 # Catch all
 /sdk/                                           @AlexGhiondea
 
-# Core
+# >>>>>>>>> Core Libraries <<<<<<<<<<
```

Comment:
I must admit that when I say this I thought it was a merge conflict. You might want to reconsider using the ">>>" format.

Reply:
Ah! YES! Good point 🙂. I'll change that! :)

Reply:
Done